src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -754,6 +773,21 @@ export function getInputType(
     }
 }
 
+function setUsage(
+    context: SdkContext,
+    source: Model | Enum,
+    target: InputModelType | InputEnumType
+) {
+    const sourceUsage = getUsageOverride(context, source);
+    if (sourceUsage === UsageFlags.Input) {
+        target.Usage = Usage.Input;
+    } else if (sourceUsage === UsageFlags.Output) {
+        target.Usage = Usage.Output;
+    } else if (sourceUsage === (UsageFlags.Input | UsageFlags.Output)) {
```

Comment:
Is there a `Usageflag.InputOutput` or `Usage.RoundTrip`?

Reply:
No, `UsageFlags.Input | UsageFlags.Output` is by design